### DISCUSSION TITLE
Questions about `pdm init --copier`

### DISCUSSION THREAD

**Discussion by astrojuanlu** [pdm-project/pdm#2992] [Q&A] [answered]
- What's the difference between `pdm init --copier ...` and `copier copy ...`? Does using the former give any advantage?
- Does the template used for `pdm init` need to have any constraints? Or could it as well use `setuptools`? (Not that I _want_ to do such a thing - seeking to understand if it would be possible)

**Comment by frostming** [comment, marked as answer]
1. no difference, it's just a wrapper. the advantages are if you install pdm using the installation script or in isolated manner, copier may not be accessible via PATH

2. Yes you can use setuptools
